core/os/file_access.cpp

```diff
@@ -362,6 +362,10 @@ Vector<String> FileAccess::get_csv_line(const String &p_delim) const {
 		}
 	}
 
+	if (in_quote) {
+		WARN_PRINT("Warning: reached end of file before closing '\"' in CSV file '" + get_path() + "'.");
```

Comment:
For consistency with the other addition, use `vformat()` here too:

```suggestion
		WARN_PRINT(vformat("Warning: reached end of file before closing '\"' in CSV file '"%s"'.", get_path()));
```